examples/iam-eks-role/main.tf

```diff
@@ -30,3 +26,33 @@ module "iam_eks_role" {
     "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy"
   ]
 }
+
+##################
+# Extra resources
+##################
+
+resource "random_pet" "this" {
+  length = 2
+}
+
+module "eks" {
```

Comment:
@antonbabenko I don't really see the point of adding a whole declaration of the `terraform-aws-eks` module in here? Now it's not really clear that you can use this module completely separately. And now the use of `(random_pet.this.id)` as the cluster name is even more confusing IMO 🙂

Reply:
"All examples should ALWAYS be executable right away as-is using `terraform apply`" - this is the standard rule we have in all modules.

`cluster1` was not a known cluster name.